### ISSUE TITLE
MNT: Use USNO URLs for IERS A after ???

### ISSUE THREAD

**Issue by pllim** [astropy/astropy#9453]
**Update: The maintenance completion keeps getting delayed.** Will just have to check https://www.usno.navy.mil/USNO/astronomical-applications once in a while...

### Description

USNO is supposed to be back up online after a prolonged maintenance period after ~April 30,~ Fall 2020. We should check if we can use its services again to get IERS A data.

### Additional context

This is a follow up of #9427 and a reminder to revert #9443 when we can.

**Comment by bsipocz** [comment]
Likewise, any fixes to the urls should go back to the bugfix branch

**Comment by pllim** [comment]
This is not going to happen before 4.0.1, so please move to the next bugfix milestone when it is available. Thanks!

**Comment by bsipocz** [comment]
Current status on the webpage, so we need to bump this to the next milestone.
```
The USNO websites aa.usno.navy.mil, ad.usno.navy.mil, aristarchus.usno.navy.mil, maia.usno.navy.mil, rorf.usno.navy.mil, and toshi.usno.navy.mil are undergoing modernization efforts.  The expected completion of the work and the estimated return of service is Summer 2020.
```

**Comment by pllim** [comment]
They didn't say summer in which hemisphere... Anyway, I'll change the title to Aug...

**Comment by bsipocz** [comment]
I'm pretty sure summer is Northern hemisphere summer for the us military

**Comment by bsipocz** [comment]
I'm not sure there is any point in dragging this through the milestones, and I would even be inclined to close it as the current solution seems to be stable enough.

**Comment by pllim** [comment]
No harm keeping this open for now but if they cannot get the server up by end of 2021, maybe we'll just give up. Agree to removing milestone, thanks!

**Comment by pllim** [comment]
Resolved by #13004
